src/components/Tooltip/DemoElements.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable */
+import React, { Fragment } from 'react'
+
+export const DemoContent = (props) =>
```

Comment:
I think this is too specific to include in `octopus`, try to only include basic elements that can be used across unrelated project. Unless i'm not understanding "Demo"